nginx/development

```diff
@@ -1,11 +1,42 @@
 include /etc/nginx/conf.d/server_config;
 
+
+log_format cache_log '$remote_addr $upstream_cache_status [$time_local] '
```

Comment:
Can the `log_format` and `proxy_cache_path` section be in `nginx/server_config` rather than here?

Reply:
yes, these do not need to be here; several of the other proxy related directives could also be relocated if desired.